Vivekananda_session2_4VP20CS062_Prajwal/EmpManagement/employees/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+
+DEPARTMENT_CHOICES = (
+    ("HR", "HR"),
+    ("Engineering", "Engineering"),
+    ("Marketing", "Marketing"),
+    ("Planning", "Planning"),
+    ("Sales","Sales"),
+    ("Finance","Finance"),
+    ("Operations","Operations"),
+)
+
+
+
+
+class Employee(models.Model):
```

Comment:
@Prajwal7Amin Any reason, why you've kept default value as None/NULL, for all the fields of the table?

Reply:
@anushds Just to keep the code flexible, thought it would make easy to handle errors

Reply:
@Prajwal7Amin While it might help you to handle errors, imagine that someone clicks on add employee from the front-end and without entering any values, click on the save button.
Now you have a entry(row) in employee table whose fields are NULL.
So, when someone checks the list of employees in the organisation, they'd see a row whose all fields are NULL.
How will you explain it to them? Do you think that's correct?

Reply:
@Prajwal7Amin Alright, so what's the primary key of the table?

Reply:
@anushds Previously I have not assigned primary key, so django automatically created a new filed 'id' with auto increment and set it to primary key.
Now I have changed the primary key to emp_id.

Reply:
@Prajwal7Amin Actually id field would've been fine as primary key.
There's something called unique key. So you could've simply made emp_id field as unique, so it won't repeat.

Reply:
@anushds Ok .